site/docs/references/state-parameter-types.md

```diff
@@ -161,19 +162,19 @@ You can use several sub validators that the number type provides:
 .root:column(4),
 .root:column(6) {}
 
-/* invalid due to "multipleOf(2)" */
+/* invalid because not a "multipleOf(2)" */
 .root:column(3),
 .root:column(5) {}
 
-/* invalid due to "min(2)" and "max(6)" */
+/* invalid because of "min(2)" and "max(6)" validations */
 .root:column(1),
 .root:column(7) {}
 ```
 
-## Future Intent
+## Coming soon
```

Comment:
lets keep it "future intent" or something that doesn't say we are currently working on this - because we're not.